src/compiler/crystal/semantic/top_level_visitor.cr

```diff
@@ -673,7 +673,7 @@ class Crystal::TopLevelVisitor < Crystal::SemanticVisitor
       base_type = options[:enum_base_type]
       is_flags = options[:is_flags]
 
-      if options[:existed]
+      if existed
```

Comment:
I'd rather remove the assignment a couple lines above. There's no need for a local variable when only used once.

Reply:
done)